metagraph/src/graph/representation/succinct/boss_chunk_construct.cpp

```diff
@@ -339,41 +355,36 @@ using Encoder = common::EliasFanoEncoderBuffered<T>;
 template <typename T>
 using Decoder = common::EliasFanoDecoder<T>;
 
-/**
- * Splits #kmers by W (kmer[0]) and F (kmer[k]) into |ALPHABET\{$}|^2 chunks.
- * @tparam T_REAL k-mers over the alphabet without the sentinel character (e.g. ACGT).
- * @return names of the files with the partitioned k-mers
- */
 template <typename T_REAL>
-std::vector<std::string> split(size_t k,
-                               const std::filesystem::path &dir,
-                               const ChunkedWaitQueue<T_REAL> &kmers) {
+std::vector<std::string> split(size_t k, const std::string &chunk_fname) {
```

Comment:
I would rather update than delete the comment? This code is already quite complex and hard to follow. Helping the reader with a brief explanation of what the method does seems like a reasonable thing to do.

Reply:
done